File: 220831/b_24390_junh.swift

let input = readLine()!.split(separator: ":")
var m = Int(input[0])!
var s = Int(input[1])!

var cnt = 1

if s >= 30 {
    s -= 30
}

cnt += m/10 + m%10 + s/10

print(cnt)

File: 220831/b_24390_junh.py
M, S = map(int, input().split(':'))
time = 0
cnt = 1

if S >= 30:
    S -= 30

S = M * 60 + S
for a in (600, 60, 30, 10):
    tmp = S // a
    S -= tmp * a
    cnt += tmp
print(cnt)

